lumicks/pylake/detail/confocal.py

```diff
@@ -167,14 +171,36 @@ def from_json(cls, json_string):
         return cls(axes, json_dict["scan volume"]["center point (um)"], json_dict["scan count"])
 
 
+class ConfocalFile:
```

Comment:
This is not really a file so how about naming this something like `ConfocalFileProxy`?

Reply:
Yeah, good point. Fixing...